<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WxJumpClass</class>
 <widget class="QWidget" name="WxJumpClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WxJump</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="screenImg">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>480</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1024</width>
         <height>2000</height>
        </size>
       </property>
       <property name="text">
        <string>请稍候...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="autoJump">
       <property name="text">
        <string>跳一下</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="manualJump">
       <property name="text">
        <string>重新获取图片</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="plainText">
        <string>使用说明:
    图中的小红点分别表示棋子位值和下一目标位值.如果下一目标位值不准确，可以通过点击 重新获取图片 按钮尝试解决。

</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="WxJump.qrc"/>
 </resources>
 <connections/>
</ui>
